bigquery/bigquery_to_pydantic.py

```diff
@@ -46,7 +48,7 @@ def pydantic_model_from_bq_schema(bq_schema_filename, output_filename):
     # Long options
     long_options = ["help", "input=", "output="]
     try:
-        opts, args = getopt.getopt(sys.argv[1:], "i:o:", ["input=", "output="])
+        opts, args = getopt.getopt(sys.argv[1:], "i:o:m", ["input=", "output=", "model="])
```

Comment:
I think we need a colon here (like this: `i:o:m:`)

Reply:
changed.